File: resharper/src/resharper-unity/Psi/Cg/CgLanguage.cs
using JetBrains.Annotations;
using JetBrains.ReSharper.Psi;

namespace JetBrains.ReSharper.Plugins.Unity.Psi.Cg
{
    [LanguageDefinition(Name)]
    public class CgLanguage : KnownLanguage
    {
        public new const string Name = "CG";
        
        [CanBeNull]
        public static readonly CgLanguage Instance = null;

        public CgLanguage() : base(Name, "Cg")
        {
        }
    }
}

File: rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/ideaInterop/fileTypes/cg/CgLanguage.kt
package com.jetbrains.rider.plugins.unity.ideaInterop.fileTypes.cg

import com.jetbrains.rider.ideaInterop.fileTypes.RiderLanguageBase

object CgLanguage : RiderLanguageBase("Cg", "CG") {
    override fun isCaseSensitive(): Boolean = true
}
